docs/index-custom-title-page.html

```diff
@@ -61,6 +61,9 @@ <h2>Documentation</h2>
       </p>
       <p>
         The Perl client provides strongly typed requests and responses for all Elasticsearch APIs.
+        The end-of-life for version 8 of the elastic/elaticsearch-perl software has been announced. 
```

Comment:
I would better highlight the end-of-life message here. Maybe in a new line with bold.